.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+# Summary
+<!-- (Madantory) short summary of this PR. -->
+
+# Details
+<!-- the details in pr like some useful information -->
+
+# Reminders 
+<!-- for developers and reviewers because most of them are not checked automatically. 
+
+Ensure you have set up the pre-commit and the corresponding pre-commit hook.  
+Use `pre-commit run --from-ref origin/develop --to-ref HEAD` to ensure the branche changes compared to origin/develop are formatted through pre-commit.
+
+Also, check https://github.com/ginkgo-project/ginkgo/wiki/Contributing-guidelines especially about naming-scheme, Whitespace, Other Code Formatting not handled by ClangFormat. 
```

Comment:
```suggestion
Also, check https://github.com/ginkgo-project/ginkgo/wiki/Contributing-guidelines, especially regarding Ginkgo's naming scheme, whitespace guidance, and other code formatting not handled by ClangFormat. 
```

Reply:
Yeah, it just bothers me that it reads strangely in the text :) Especially the dash in "naming-scheme".

What about adding direct links to each section? E.g. something like

Also, check the [contributing guidelines](https://github.com/ginkgo-project/ginkgo/wiki/Contributing-guidelines), especially regarding the [naming scheme](https://github.com/ginkgo-project/ginkgo/wiki/Contributing-guidelines#naming-scheme), [whitespace use](https://github.com/ginkgo-project/ginkgo/wiki/Contributing-guidelines#whitespace), and [other code formatting not handled by ClangFormat](https://github.com/ginkgo-project/ginkgo/wiki/Contributing-guidelines#other-code-formatting-not-handled-by-clangformat).

Reply:
because it is not shown in the rich view, it gives several link url in the text

Reply:
Is that a problem?

I think we should at least be consistent -- right now, "naming-scheme" is the label for the internal link, while "Whitespace" (capitalized) and "Other Code Formatting not handled by ClangFormat" are from the header text (the text from the wiki comes from CONTRIBUTING.md, right?).

What about

```
Also, check https://github.com/ginkgo-project/ginkgo/wiki/Contributing-guidelines, particularly the `Naming scheme`, `Whitespace`, and `Other Code Formatting not handled by ClangFormat` sections.
```